src/videouplinkbandwidthpolicy/VideoUplinkBandwidthPolicy.ts

```diff
@@ -57,4 +58,16 @@ export default interface VideoUplinkBandwidthPolicy {
    * Returns the selected [[MediaTrackConstraints]] to update
    */
   chooseMediaTrackConstraints(): MediaTrackConstraints;
+
+  /**
+   * Set a reference to the current transceiver controller
+   * @param {TransceiverController} transceiverController - The transceiver controller
+   */
+  setTransceiverController?(transceiverController: TransceiverController): void;
+
+  /**
+   * Update the transceiver controller
+   * @param {TransceiverController} transceiverController - The transceiver controller
+   */
+  updateTransceiverController?(transceiverController: TransceiverController): void;
```

Comment:
These need to be a lot more thorough.

* When will these methods be called?
* What will be called when the call ends, so the policy can tear down its state and unsubscribe from things? 
* What is the policy's responsibility if it implements these, and which other methods does it obsolete? (We will no longer set bandwidth for you, right?)
* What happens if Unified Plan is not enabled?

Reply:
Are these methods allowed to throw?

Reply:
Why do both methods take a transceiver controller? Can calls to `updateTC` be made with a different TC than `setTC`?

Reply:
Got issue with git branch this morning so I think i did not get my changes to remove the transceiver param in `updateTC` in.

Reply:
Updated the interface documentation.